assets/javascripts/test_result.js

```diff
@@ -600,17 +600,21 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
 }
 
 function githashToLink(value, repo) {
-  var hashes_regex = /(\b[0-9a-f]{9}\b)/gms;
-  var githashes = value.match(hashes_regex);
-  var oneLineStatRegex = /\<a[^]+/gms;
-  var statRegex = /(\<a.+?)(?=(\<a))/gms;
-  if (!Array.isArray(githashes)) return;
-  for (let i = 0; i < githashes.length; i++) {
-    value = value.replace(githashes[i], githashes[i].link(repo + githashes[i]));
-    if (githashes.length == 1) return value.match(oneLineStatRegex);
+  const log_items = value.split(/(?=^[0-9a-f])/gm);
```

Comment:
It would be generally nice to use JS conventions (e.g. logItems). But not important enough to change everything now.